phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerEngineSettingsTab.java

```diff
@@ -60,14 +60,31 @@ private void populate() {
         final GridPane displayContentPane = new GridPane();
         displayContentPane.getStyleClass().add("grid");
 
+        int row = 0;
+        for (EngineSetting setting : engineSettings) {
+            final ComboBox<String> comboBox = new ComboBox<>();
+            comboBox.setMaxWidth(Double.MAX_VALUE);
+            comboBox.setValue(setting.getCurrentOption());
```

Comment:
If we want to we could use `tr()` here.
The question is how do we collect the different options into the translation related files?

Reply:
I was thinking do solve the issue a bit differently: Instead of just passing a `String[]` we could use a `SettingValue` object which has a `value` and a `text`. Only the `text` would be translated.
Alternatively, we could also pass the index of the selected combobox item instead of the String to `setOption`.
Not sure what's the best solution.

Reply:
I think it is important to implement this with the goal that writing a new `EngineSetting` is as easy as possible.

I think if we use your first option it will require that each settings defines multiple `SettingValue` objects, which need to be returned.
The second idea sounds good to me.